src/Microsoft.Diagnostics.Runtime/src/DataReaders/Core/CoreDumpReader.cs

```diff
@@ -171,14 +165,10 @@ private IList<CoreModuleInfo> EnumerateAllModules()
                 // memory range overlaps with actual modules.
                 ulong interpreter = _core.GetAuxvValue(ElfAuxvType.Base);
 
-                _modules = new List<CoreModuleInfo>();
```

Comment:
I think CoreModuleInfo was introduced in your last change unless my memory is wrong.  Is CoreModuleInfo still used or is it dead code?  If dead can you remove it?  (If it's still used then it's fine to leave it, I just wanted to make sure we still need the class and aren't leaving behind unused code.)